integration_tests/e2e/induction/updateCheckYourAnswersChangeLinks.cy.ts

```diff
@@ -147,32 +154,26 @@ context(`Change links on the Check Your Answers page when updating an Induction`
       .hasEducationalQualifications(['Physics'])
   })
 
-  /* TODO - RR-736 finish the implementation of this test by clicking all Change links:
-   * Hoping To Work On Release (perhaps not do this one as it will turn the induction back into a short question set!)
-   * Educational Qualifications - Done
-   * Highest Level of Education - Done
-   * Other Training
-   * Worked Before
-   * Work history
-   * Type of work interested in
-   * Particular job roles
-   * Personal Skills
-   * Personal Interests
-   * Factors affecting ability to work
-   */
   it('should support all Change links on a Long Question Set Induction', () => {
     // Given
     cy.updateLongQuestionSetInductionToArriveOnCheckYourAnswers(prisonNumber)
-    const checkYourAnswersPage = Page.verifyOnPage(CheckYourAnswersPage) // eslint-disable-line @typescript-eslint/no-unused-vars
+    Page.verifyOnPage(CheckYourAnswersPage)
 
     // When
+    // Change Hoping To Work On Release (Yes only)
```

Comment:
You don't need this change (changing Hoping To Work On Release to Yes) because its's already set to Yes by virtue of `cy.updateLongQuestionSetInductionToArriveOnCheckYourAnswers(prisonNumber)` on line 159. Setting it to Yes makes it a long question set Induction, but it's already a long question set induction.

We already have the tests that change this value and change the question set from short to long (and vica versa) in `updateInductionQuestionSet.cy.ts` 👍

Reply:
If there's no value in it I can remove, but I added it to check that the redirect back to check answers works correctly.

It verifies that if Yes -> Yes it redirects straight back to check your answers rather than initiating the long question set journey – that should only happen if going from No -> Yes.

Reply:
It will slow the test down a little (as it's a page rendering and submission), but if you want to keep it thats fine.
As you're discovering the whole Induction journey and the various flow changes within it are complex, so if it helps you (and others in the future) understand it then it serves its purpose well 👍 😁